src/Zikula/CoreBundle/Composer/Scanner.php

```diff
@@ -73,18 +74,18 @@ public function decode(string $jsonFilePath)
     {
         $base = str_replace('\\', '/', dirname($jsonFilePath));
         $srcPath = dirname(__DIR__, 4) . '/src/';
-        $base = mb_substr($base, mb_strlen($srcPath));
+        $base = s($base)->slice(mb_strlen($srcPath));
```

Comment:
now here you do not cast `(string)`. It seems this should be consistent.

Reply:
Spacing for casts were not consistent before already. Improved that in 7a1aa6ab9130a1ffde0c5c997d5fe5c005ea7cac

Reply:
Added `__toString()` call in bd77d26e3d554ba2894d9e53b90ebe3ad97b84b8